pages/try.js

```diff
@@ -1,12 +1,17 @@
 import dynamic from "next/dynamic";
 
-const Try = dynamic(() => import("src/Try.mjs"), {
+export { getStaticProps } from "src/Try.mjs";
+import Try from "src/Try.mjs";
+
+const Playground = dynamic(() => import("src/Playground.mjs"), {
   ssr: false,
-  //loading: () => <div> Loading... </div>
-});
+  loading: () => <span>Loading...</span>,
+})
```

Comment:
Not for now, but this should be possible to move into ReScript now that we have dynamic imports, right?

Reply:
yup